docs/src/user/manual/explore/filters.md

```diff
@@ -13,6 +13,9 @@ context_id: GD6YdjhPx43u8TKLi3eNNE
 ---
 
 # Filter Side Panel Documentation
+# Map Tab Filter Panel Overview
+
+> **Disclaimer:** Please read the following information carefully. The screenshots may vary slightly due to various user roles on the platform.
 
 The filter side panel allows users to refine and narrow down data based on specific criteria. There are different types of filters are available for customisation.
 The filter side panel enhances the user experience by allowing them to focus on specific subsets of data based on their preferences, making it easier to analyse and work with the dataset effectively.
```

Comment:
There should only be one level 1 heading per file, and also maybe rearrange things like:
```suggestion
# Filter Side Panel Documentation

> **Disclaimer:** Please read the following information carefully. The screenshots may vary slightly due to various user roles on the platform.

The filter side panel allows users to refine and narrow down data based on specific criteria. There are different types of filters are available for customisation.

## Map Tab Filter Panel Overview

The filter side panel enhances the user experience by allowing them to focus on specific subsets of data based on their preferences, making it easier to analyse and work with the dataset effectively.
```